File: modulo1/segmento3/apunte4/ejercicio2/ejercicio2.web/ejercicio2.js
function validar(){
	const caracter = prompt('Ingrese un caracter para determinar si es vocal, consonante o un numero: ');
				
		switch (caracter){
			case'1':
			case'2':
			case'3':
			case'4':
			case'5':
			case'6':
			case'7':
			case'8':
			case'9':
			case'0':

				alert ('Es Numero');
			 	break;

			case'a':
			case'e':
			case'i':
			case'o':
			case'u':
				alert('Es una Vocal');
				break;

			case'b':
			case'c':
			case'd':
			case'f':
			case'g':
			case'h':
			case'j':
			case'k':
			case'l':
			case'm':
			case'n':
			case'p':
			case'q':
			case'r':
			case's':
			case't':
			case'v':
			case'w':
			case'x':
			case'y':
			case'z':
				alert('Es una Consonante');
				break;
			
			default:
				alert('Error, UD ingreso un symbolo');
				break;
		}
			
			
}

File: modulo1/segmento3/apunte4/ejercicio2/ejercicio2.java
import java.util.Scanner;

public class ejercicio2{
	public static void main(String args[]){
		Scanner keyboard = new Scanner(System.in);

			System.out.print("Ingrese un caracter para ver si es, vocal, consonante o numero: ");
			String caracter = keyboard.nextLine();

			switch(caracter){
				case"1":
				case"2":
				case"3":
				case"4":
				case"5":
				case"6":
				case"7":
				case"8":
				case"9":
				case"0":
					System.out.println("Es Numero");
					break;

				case"a":
				case"e":
				case"i":
				case"o":
				case"u":
					System.out.println("Es Vocal");
					break;
				
				case"b":
				case"c":
				case"d":
				case"f":
				case"g":
				case"h":
				case"j":
				case"k":
				case"l":
				case"m":
				case"n":
				case"p":
				case"q":
				case"r":
				case"s":
				case"t":
				case"v":
				case"w":
				case"x":
				case"y":
				case"z":
					System.out.println("Es Consonante:");
					break;
				
				default:
					System.out.println("Error ustes ingreso un symbolo");
					break;
			}
	}
}

				
